### PR TITLE
Iteration/fiets subsidy 3

### PR DESCRIPTION
**Pull request by aatauil**
See miro board climate subsidy version 1

### CODE REVIEW

**Review by Windvis — COMMENTED** [review]
I only took a look at the code itself, (not the business logic since I can't evaluate that) and noticed some small things that could be improved.

**Comment by aatauil** [comment]
@Windvis All changes have been made. Claire already reviewed the business logic and found no issues. Could you merge it if the changes are ok? Thanksss

File: addon/components/custom-subsidy-form-fields/climate-subsidy-costs-table/table-row-burgemeesters.hbs [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
     {{yield (hash actionDescription = (component 'custom-subsidy-form-fields/climate-subsidy-costs-table/table-cell-data')) }}
-  </td>
+  </th>
   <td>
-    {{convert-to-currency this.costPerUnit}} per actieplan
+    <AuInput
+      disabled={{@disabled}}
+      type="number"
+      lang="nl-BE"
+      @width={{"block"}}
```
[the remark this answers is not in the document]
Comment (aatauil) [reply addon/components/custom-subsidy-form-fields/climate-subsidy-costs-table/table-row-burgemeesters.hbs]: Done, all occurrences in the whole addon of this have been changed

File: addon/components/custom-subsidy-form-fields/climate-subsidy-costs-table/table-row-burgemeesters.hbs [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
+      @value={{this.costPerUnit}}
+      class={{if this.costPerUnitErrors "au-c-input--error"}}
```
[the remark this answers is not in the document]
Comment (aatauil) [reply addon/components/custom-subsidy-form-fields/climate-subsidy-costs-table/table-row-burgemeesters.hbs]: Indeed @error is better, all done

File: addon/components/custom-subsidy-form-fields/climate-subsidy-costs-table/table-row-burgemeesters.hbs [hunk 3]
```diff
@@ -1,9 +1,23 @@
...
+      {{on "blur" (fn this.update)}}
```
[the remark this answers is not in the document]
Comment (aatauil) [reply addon/components/custom-subsidy-form-fields/climate-subsidy-costs-table/table-row-burgemeesters.hbs]: Changed all occurences of it

File: addon/components/custom-subsidy-form-fields/climate-subsidy-costs-table/table-row-burgemeesters.hbs [hunk 4]
```diff
@@ -1,9 +1,23 @@
...
+    />
+    per actieplan
+    {{#each this.costPerUnitErrors as |error|}}
+      <AuHelpText @error={{"error"}}>{{error.message}}</AuHelpText>
```
[the remark this answers is not in the document]
Comment (aatauil) [reply addon/components/custom-subsidy-form-fields/climate-subsidy-costs-table/table-row-burgemeesters.hbs]: Same as the @width one, all occurrences over the whole addon have been changed

File: addon/components/custom-subsidy-form-fields/climate-subsidy-costs-table/table-row-burgemeesters.js [hunk 5]
```diff
@@ -50,6 +51,15 @@ export default class CustomSubsidyFormFieldsClimateSubsidyCostsTableTableRowBurg
     return this.args.populationCount;
   }
 
+  get indication(){
+    const cost = 0.15 * this.population;
```
[the remark this answers is not in the document]
Comment (aatauil) [reply addon/components/custom-subsidy-form-fields/climate-subsidy-costs-table/table-row-burgemeesters.js]: Added a clear name for the magic number. Dont know if it is the best variable name but it definitely describes what it is

File: addon/components/custom-subsidy-form-fields/climate-subsidy-costs-table/edit.hbs [hunk 6]
```diff
@@ -102,7 +88,8 @@
           Steun voor het planten van bomen op publiek domein
         </row.actionDescription>
         <row.costPerUnitDescription>
-          per boom
+          per boom <br>
+          <span style="font-style: italic;">Indicatie: € 150,00</span>
```
[the remark this answers is not in the document]
Comment (aatauil) [reply addon/components/custom-subsidy-form-fields/climate-subsidy-costs-table/edit.hbs]: Found a perfect fit in the appuniversum docs for it

**Comment by Windvis** [comment]
Looks good! The branch seems to have merge conflicts though, so it can't be merged.

**Comment by Windvis** [comment]
Also, I just noticed it targets "master", should it be "development"?

**Pull request merged**